eplaunch/interface/widget_file_list.py

```diff
@@ -99,34 +99,30 @@ def __init__(self, parent, on_selection_changed: Optional[Callable[[List[str]],
         scrollbar.grid(row=0, column=1, sticky=NS)
 
 
-def printer(selection: List[str]) -> None:
-    print(selection)
-
-
-counter = 0
-
-
-def set_columns():
-    global counter
-    counter += 1
-    if counter == 1:
-        file_listing.tree.set_new_columns(['hi', 'world'])
-    elif counter == 2:
-        file_listing.tree.set_new_columns(['foo'])
-    elif counter == 3:
-        file_listing.tree.set_new_columns(['hello', 'world', 'bar'])
-    elif counter == 4:
-        file_listing.tree.set_new_columns()
-    elif counter == 5:
-        file_listing.tree.set_new_columns(['edwin', 'lee'])
-
-
 if __name__ == "__main__":
     from tkinter import Button, Tk
 
     root = Tk()
     root.title('File Listing Widget Demo')
 
+    def printer(selection: List[str]) -> None:
```

Comment:
These are only changes in the test script at the bottom of the file, no functional GUI changes above.